torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -202,8 +203,8 @@ void FusionExecutor::compileFusion(
     kernel->print();
   }
 
-  const auto kernel_code = codegen::generateCudaKernel(kernel, kernelName());
-  const auto structured_code = getStructuredCode(kernel_code);
+  const auto kernel_code_ = codegen::generateCudaKernel(kernel, kernelName());
```

Comment:
nitpick, trailing underscore on non class data member. Saw this in a few other places.

Reply:
Wait I think you are using member, ~const auto~

Reply:
was supposed to be member, forgot to remove const auto.